src/librustc_driver/lib.rs

```diff
@@ -1239,11 +1239,19 @@ pub fn init_env_logger(env: &str) {
         Ok(s) if s.is_empty() => return,
         Ok(_) => {}
     }
-    let builder = tracing_subscriber::FmtSubscriber::builder();
-
-    let builder = builder.with_env_filter(tracing_subscriber::EnvFilter::from_env(env));
-
-    builder.init()
+    let filter = tracing_subscriber::EnvFilter::from_env(env);
+    let layer = tracing_tree::HierarchicalLayer::default()
+        .with_indent_lines(true)
+        .with_ansi(true)
+        .with_targets(true)
+        .with_thread_ids(true)
+        .with_thread_names(true)
+        .with_wraparound(10)
+        .with_indent_amount(2);
```

Comment:
Oh wow these are some global decisions being made here.^^ Maybe that should be a separate PR, it's not really related to the interpreter stack.

I'd probably set wrap-around to 25 or so. Most screens today are fairly wide.
Everything else is the default I assume? Might be worth saying that in a comment.

Reply:
> Oh wow these are some global decisions being made here.^^ Maybe that should be a separate PR, it's not really related to the interpreter stack.

We need something here, otherwise there won't be any wrapping or indent. I don't really want to drag out this PR even longer. If ppl disagree with the default settings they can open issues.

> I'd probably set wrap-around to 25 or so. Most screens today are fairly wide.

it's counted in indentations, not in chars, so that's would be 50 chars, which is half your terminal. Anyway, we can fine tune this later or make it configurable via env vars, or...

> Everything else is the default I assume?

No, everything being configured is not the default.

Reply:
Uh, but then why so many changes from the default?^^

> it's counted in indentations, not in chars, so that's would be 50 chars, which is half your terminal. Anyway, we can fine tune this later or make it configurable via env vars, or...

Fair.